githubscript/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.8
+
+RUN groupadd --gid 10001 app && \
+    useradd -g app --uid 10001 --shell /usr/sbin/nologin --create-home --home-dir /app app
+
+RUN apt-get update \
+ && apt-get install -y default-jdk \
+ && apt-get clean \
+ && ln -s /app/docker.d/healthcheck /bin/healthcheck
+
+USER app
+WORKDIR /app
+
+COPY . /app
+
+RUN python -m venv /app \
```

Comment:
I think the k8s-image task is trying to find `scriptworker_client` in pypi; it's not there.
You may need to add

```
 && cd /app/scriptworker_client \
 && /app/bin/pip install -r requirements/base.txt \
 && /app/bin/pip install . \
```

here like in treescript's dockerfile.